dbms/src/Interpreters/InterpreterSystemQuery.cpp

```diff
@@ -289,7 +290,7 @@ void InterpreterSystemQuery::restartReplicas(Context & system_context)
     if (replica_names.empty())
         return;
 
-    ThreadPool pool(std::min(getNumberOfPhysicalCPUCores(), replica_names.size()));
+    ThreadPool pool(std::min(size_t(getNumberOfPhysicalCPUCores()), replica_names.size()));
```

Comment:
C-style cast doesn't look good - may be `std::min<size_t>(…)` is better?

Reply:
std::min takes both arguments by references.